mender-update/standalone/standalone.cpp

```diff
@@ -48,7 +49,7 @@ ExpectedOptionalData LoadData(database::KeyValueDatabase &db) {
 		}
 	}
 
-	auto exp_json = json::Load(string(exp_bytes.value().begin(), exp_bytes.value().end()));
+	auto exp_json = json::Load(common::StringFromByteVector(exp_bytes.value()));
```

Comment:
Hmm, honestly, me too. Typical mysterious preprocessor effects. Too bad that [C++ Modules](https://en.cppreference.com/w/cpp/language/modules) are too new to be used, because they would improve this problem.

In any case, I added it now, since it is better to have it there.